generator-tests/aa-million-messages.txt

```diff
@@ -0,0 +1,6 @@
+ff01ffffffa00101010101010101010101010101010101010101010101010101010101010101ffff02ffff01ff02ff02ffff04ff02ffff04ff05ff80808080ffff04ffff01ff02ffff03ff05ffff01ff04ffff01ff42ff24ff8568656c6c6fffa0010101010101010101010101010101010101010101010101010101010101010180ffff04ffff01ff43ff24ff8568656c6c6fffa0010101010101010101010101010101010101010101010101010101010101010180ffff02ff02ffff04ff02ffff04ffff11ff05ffff010180ff808080808080ff8080ff0180ff018080ff7bffff832d651880808080
```

Comment:
This is a generator with this puzzle:
```
(mod (N)
    (defun another_send (n)
        (if n
            (c (list 66 36 "hello" 0x0101010101010101010101010101010101010101010101010101010101010101)
                (c (list 67 36 "hello" 0x0101010101010101010101010101010101010101010101010101010101010101)
                    (another_send (- n 1))
                )
            )
            ()
        )
    )

    (another_send N)
)
```

With the number of iterations, `N`, being set to 2975000